stack/docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+name: default-services
+
+services:
+  postgres:
+    container_name: postgres
+    image: postgres:16.0-alpine3.18
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=admin
+      - POSTGRES_PASSWORD=admin
+    ports:
+      - "5432:5432"
+
+  pg-admin:
+    container_name: pg-admin
+    image: dpage/pgadmin4
+    environment:
+      - PGADMIN_DEFAULT_EMAIL=example@example.com
+      - PGADMIN_DEFAULT_PASSWORD=admin
+    ports:
+      - "5050:80"
+
+  keycloak:
+    container_name: keycloak
+    image: quay.io/keycloak/keycloak:20.0.3
+    command:
+      - start-dev --http-relative-path /auth
+    depends_on:
+      - db-postgres-keycloak
+    environment:
+      - KEYCLOAK_ADMIN=${KEYCLOAK_ADMIN}
+      - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD}
+      - KC_HOSTNAME_STRICT=false
+      - DB=postgres
+      - DB_URL=jdbc:postgresql://db-postgres-keycloak:5432/${KEYCLOAK_DB}
+      - DB_USERNAME=${KEYCLOAK_DB_USER}
+      - DB_PASSWORD=${KEYCLOAK_DB_PASSWORD}
+    ports:
+      - "8080:8080"
+    networks:
+      - keycloak
+
+  db-postgres-keycloak:
+    container_name: db-postgres-keycloak
+    image: postgres:16.3-alpine3.18
+    environment:
+      - POSTGRES_DB=${KEYCLOAK_DB}
+      - POSTGRES_USER=${KEYCLOAK_DB_USER}
+      - POSTGRES_PASSWORD=${KEYCLOAK_DB_PASSWORD}
+    networks:
+      - keycloak
+
+  init-keycloak:
+    container_name: init-keycloak
+    image: klg71/keycloakmigration:0.2.55
+    depends_on:
+      - keycloak
+    env_file:
+      - '.env'
+    environment:
+      ADMIN_USER: ${KEYCLOAK_ADMIN}
+      ADMIN_PASSWORD: ${KEYCLOAK_ADMIN_PASSWORD}
+      BASEURL: ${SSO_BASE_URL}
+      WAIT_FOR_KEYCLOAK: 'true'
+      KEYCLOAK_CHANGELOG: /migration/keycloak-changelog.yml
+    volumes:
+      - './keycloak/migration:/migration'
+    networks:
+      - keycloak
+
+  refarch-gateway:
+    container_name: refarch-gateway
+    image: ghcr.io/it-at-m/refarch/refarch-gateway:latest
```

Comment:
I highly discourage the use of latest image.
We as template providers should make sure our templates work with a specific version if the API gateway. If we set latest here, bumping the API-Gateway of the stack is no longer under our control. This might lead to side effects.
I suggest we manually bump API-Gateway versions as its a external dependency just like db, mail, etc.